scripts/quests/windurst/Class_Reunion.lua

```diff
@@ -0,0 +1,207 @@
+            return status == QUEST_ACCEPTED
+        end,
+
+        --Section: Quest Accepted
+
+        [xi.zone.WINDURST_WALLS] =
+        {
+
+            ['Koru-Moru'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 1 then
+                        return quest:progressEvent(412)
+                    elseif questProgress == 2 then
+                        return quest:progressEvent(414)
+                    end
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.ASTRAGALOS) then
+                        return quest:progressEvent(407)
+                    end
+                end,
+            },
+
+            ['Shantotto'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 3 then
+                        return quest:event(409)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [412] = function(player, csid, option, npc)
+                    player:delKeyItem(player, xi.ki.CARBUNCLES_TEAR)
+                    player:messageSpecial(ID.text.LOST_KEYITEM, xi.ki.CARBUNCLES_TEAR)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [407] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 3)
+                end,
+
+                [410] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+            }
+        },
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Fuepepe'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 3 then
+                        return quest:event(817)
+                    end
+                end,
+            },
+
+            ['Furakku-Norakku'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 3 then
+                        return quest:event(816)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Gulmama'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 3 then
+                        return quest:progressEvent(713)
+                    elseif questProgress == 4 and not player:hasItem(xi.items.ICE_PENDULUM) then
+                        return quest:event(712)
```

Comment:
```suggestion
                        return quest:progressEvent(712)
```

Reply:
also, I noticed you took out the reference to the pendulums in these event calls, make sure you put them back in.